krun.py

```diff
@@ -277,9 +307,27 @@ def main():
     config = util.read_config(config_file)
     out_file = util.output_name(config_file)
 
+    mail_recipients = config.get("MAIL_TO", [])
```

Comment:
It is indeed optional. If the directive is missing in the config file, then it defaults to `[]`, which means mail no one. See mailer.py:55.